webBrow/index.js

```diff
@@ -0,0 +1,326 @@
+        display = 0
+      }
+      if(display == 0){
+        display = ''
+      }
+      display += num
+    }
+    return calculator.displayNode.innerHTML = display
+  }
+
+  Calculator.prototype.getOperand = function(symbol){
+    var calculator = this
+    if(calculator.operand !== ''){
+      calculator.operand = ''
+      calculator.first = 0
+    }
+    switch(symbol){
+      case '+':
+        calculator.operand = '+'
+        break
+      case '-':
+        calculator.operand = '-'
+        break
+      case 'X':
+        calculator.operand = '*'
+        break
+      case '*':
+        calculator.operand = '*'
+        break
+      case '÷':
+        calculator.operand = '/'
+        break
+      case '/':
+        calculator.operand = '/'
+        break
+    }
+    calculator.firstValueAdded = true
+  }
+
+  Calculator.prototype.calculate = function() {
+    var calculator = this
+    if(calculator.first !==0){
+      calculator.second = calculator.displayNode.innerHTML
+      calculator.secondValueAdded = true
+      if(calculator.second !== 0){
+        switch(calculator.operand){
+          case '+':
+            calculator.result = parseFloat(calculator.first) + parseFloat(calculator.second)
+            break
+          case '-':
+            calculator.result = parseFloat(calculator.first) - parseFloat(calculator.second)
+            break
+          case '*':
+            calculator.result = parseFloat(calculator.first) * parseFloat(calculator.second)
+            break
+          case '/':
+            calculator.result = parseFloat(calculator.first) / parseFloat(calculator.second)
+            break
+        }
+        calculator.displayNode.innerHTML = calculator.result
+        calculator.changeFontSize()
+      }
+    }
+  }
+
+  Calculator.prototype.allClear = function() {
+    var calculator = this
+    calculator.displayNode.innerHTML = 0
+    calculator.displayNode.style.fontSize = "50px"
+    calculator.displayNode.style.fontWeight = "lighter"
+    calculator.first = 0
+    calculator.second = 0
+    calculator.operand = ''
+    calculator.result = ''
+    calculator.firstValueAdded = false
+    calculator.secondValueAdded = false
+  }
+
+  Calculator.prototype.flipSign = function() {
+    var calculator = this.displayNode.innerText
+    var flip = calculator.displayNode.innerText.indexOf('-')
+    if(toFlip > 0){
+      calculator = '-' + toFlip
+    }else{
+      calculator = calculator.slice(flip+1)
+    }
+  }
+
+  Calculator.prototype.percent = function() {
+    var calculator = this
+    var perc = calculator.displayNode.innerHTML
+    var removeIndex = perc.indexOf('%')
+    var toBeCalculated = perc.replace(/\% ?/g, "")
+    perc = (toBeCalculated / 100).toFixed(2)
+    return calculator.displayNode.innerHTML = perc
+
+  }
+
+  Calculator.prototype.isOperand = function(sym){
+    return ['+', '-', '/', '*'].includes(sym)
+  }
+  Calculator.prototype.assignClass = function(eventKey){
```

Comment:
Looking at this function, it looks like it is adding the effect of the click. I would look at the css property `:active` to see if I could use that instead